bindings/wasm/examples/public/examples.js

```diff
@@ -502,7 +502,3 @@ for (const [func, code] of Object.entries(examples.functions)) {
       func.replace(/([a-z])([A-Z])/g, '$1 $2');  // Add spaces between words
   examples.functionBodies.set(name, body);
 };
-
-if (typeof self !== 'undefined') {
```

Comment:
Was this something you added in your recent commit that I missed again? Will add back.

Reply:
yeah, thanks.